app/assets/index.html

```diff
@@ -22,6 +24,9 @@
     </script>
   </head>
   <body>
+    <noscript>
+      <p style="text-align: center;">Please enable JavaScript to use Yugioh Calculator.</p>
```

Comment:
A "reload" button won't work if JS is disabled, but yes, I'll add prettier and more-consistent styling.